### DISCUSSION TITLE
How to convert a float32 to a pgtype.Numeric

### DISCUSSION THREAD

**Discussion by oliviernguyenquoc** [jackc/pgx#1854] [Q&A]
How to convert an unknow float32 value to a pgtype.Numeric ?

Tests in the code are not clear due to the structure of pgtype.Numeric:

```
type Numeric struct {
	Int              *big.Int
	Exp              int32
	NaN              bool
	InfinityModifier InfinityModifier
	Valid            bool
}
```

**Comment by gabrieldarruiz** [comment]
its not clear since it returns a float8 number. Im having the same issue here
